btax/run_btax.py

```diff
@@ -68,10 +78,44 @@ def run_btax(**user_params):
     return output_by_asset, output_by_industry
 
 
+def _diff_two_tables(df1, df2):
+    assert tuple(df1.columns) == tuple(df2.columns)
+    diffs = OrderedDict()
+    for c in df1.columns:
+        example = getattr(df1, c).iloc[0]
+        can_diff = isinstance(example, numbers.Number)
+        if can_diff:
+            diffs[c] = getattr(df1, c) - getattr(df2, c)
+        else:
+            diffs[c] = getattr(df1, c)
+    return pd.DataFrame(diffs)
+
+
+def run_btax_with_baseline_delta(**user_params):
+    base_output_by_asset, base_output_by_industry = run_btax()
```

Comment:
@PeterDSteinberg , even in the baseline case, we need to pass in the user defined parameters, because they may change the economic assumptions (even though inputs have changed, we still call this the "baseline" since the tax policies are the same).  Thus we'll always run a baseline and reform using the same economics, with all differences being driven changes to tax policy.